llvm/test/tools/dsymutil/X86/dwarf5-many-include-directories.ll

```diff
@@ -0,0 +1,2631 @@
+; RUN: rm -rf %t && mkdir -p %t
+; RUN: %llc_dwarf -o %t/all.o -filetype=obj %s
+; Dummy debug map to get the results. Bare object doesn't seem to work
+; RUN: echo "---" > %t/debug.map
+; RUN: echo "triple: 'x86_64-apple-darwin'" >> %t/debug.map
+; RUN: echo "objects:" >> %t/debug.map
+; RUN: echo "  - filename: %t/all.o" >> %t/debug.map
+; RUN: echo "    symbols:" >> %t/debug.map
+; RUN: echo "      - { sym: _all, objAddr: 0x0, binAddr: 0x0, size: 0x0 }" >> %t/debug.map
+; RUN: echo "..." >> %t/debug.map
```

Comment:
`split-file` would allow you to write `debug.map` inline in the file without using `echo`. I've used `sed` to write the temporary file path into these files before.

https://llvm.org/docs/TestingGuide.html#extra-files